deliver/lib/deliver/languages.rb

```diff
@@ -1,7 +1,126 @@
+require_relative 'upload_metadata'
```

Comment:
Technically speaking, this doesn't look good as `languages.rb` and `upload_metadata.rb` depends on each other. This can potentially cause a circular dependency.  Maybe we can keep `Deliver::PrepareLanguages` in `upload_metadata.rb` for now?